app/helpers/games_helper.rb

```diff
@@ -1,2 +1,12 @@
 module GamesHelper
+
+  # def test
+  #   "This is a test"    
+  # end
+  def render_piece(x, y)
+    piece = @game.pieces.find_by(position_x, position_y)
```

Comment:
Here's the line dropping the error. Says position_x is an undefined local variable. I'm assuming it would say the same about position_y if it got that far.

Reply:
See the `find_by` docs. See if you can find what's wrong here.

http://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-find_by